theme/functions.php

```diff
@@ -76,7 +76,6 @@ function configure_menu() {
 function election_data_init() {
 	register_nav_menu('header-menu', __( 'Header Menu' ) );
 	add_theme_support( 'custom-header' );
-    add_theme_support( 'post-thumbnails' );
```

Comment:
This shouldn't be removed as it's required for candidates and parties to have images.